### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by rajgodwin**
Added details for 2.4.2 till 2.5 version releases

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ This page provides the latest Teams PowerShell change log for both General Avail
 
 | Date | Version | Updates |
 |------- | -------------------- | ------------------------------ |
+| August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only)</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only).</li>|
```

File: Teams/teams-powershell-release-notes.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
+| July 2021 | [2.4.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.1-preview) |<li>Grant cmdlets changes now available.</li><li>New Voice related cmdlets are released.</li><li>Removal of certificate thumbprint authentication for -Cs* cmdlets.</li><li>Logging fix for logging files of all cmdlets.</li><li>Fixes issues with *TeamChannelUser cmdlets</li>|
+| June 2021 | [2.4.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.0-preview) |<li>Preview only release of modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only), Get-CsOnlineDialInConferencingLanguagesSupported, and Import-CsOnlineAudioFile.</li><li>Modernized versions of Get-CsOnlineDialInConferencingLanguagesSupported and Import-CsOnlineAudioFile are expected to work similar/same to their remoting counter parts.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:31]:
```suggestion
| August 2021 | [2.5.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.0) |<li>Updates for AccessToken login with Connect-MicrosoftTeams.</li><li>Fixes for Interactive login of Connect-MicrosoftTeams in Cloudshell.</li><li>Improvements to New-Team cmdlet for team creation scenarios.</li><li>TeamsUnassignedNumberTreatment cmdlets are now available.</li><li>Get-CsCsOnlineDialInConferencingBridge and Set-CsOnlineDialInConferencingBridge.</li><li>Releases modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only).</li>|
| July 2021 | [2.4.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.1-preview) |<li>Grant cmdlets changes now available.</li><li>New Voice related cmdlets are released.</li><li>Removal of certificate thumbprint authentication for -Cs* cmdlets.</li><li>Logging fix for logging files of all cmdlets.</li><li>Fixes issues with *TeamChannelUser cmdlets.</li>|
| June 2021 | [2.4.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.0-preview) |<li>Preview only release of modernized versions of Get-CsTenant, Get-CsOnlineUser (with -identity parameter only), Get-CsOnlineDialInConferencingLanguagesSupported, and Import-CsOnlineAudioFile.</li><li>Modernized versions of Get-CsOnlineDialInConferencingLanguagesSupported and Import-CsOnlineAudioFile are expected to work similar/same to their remoting counterparts.
```

File: Teams/teams-powershell-release-notes.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
+  - Modernized versions of Get-CsTenant and Get-CsOnlineUser (when run with -identity parameter) don't emit deprecated properties.
+  - Modernized versions of Get-CsTenant and Get-CsOnlineUser (when run with -identity parameter) have some formatting changes when compared to their remoting counter parts.</li><li>Releases [Get|Set|Grant|New|Remove]-CsTeamsAudioConferencingPolicy cmdlets.</li><li>Releases Get-CsOnlineAudioFile and Remove-CsOnlineAudioFile cmdlets.</li><li>Set-TeamTargetingHierarchy, Remove-TeamTargetingHierarchy, Get-TeamTargetingHierarchyStatus are now available for GCC customers.</li><li>Fixes the endpoint called by the Get-TeamTargetingHierarchyStatus command.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:33]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rajgodwin, kindly commit the suggested changes (see rich diff view). Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**